packages/taquito-rpc/test/taquito-rpc.spec.ts

```diff
@@ -3911,11 +3919,13 @@ describe('RpcClient test', () => {
           },
         ],
       });
-      const response = await client.runOperation(testData as any);
+      const response = await client.runOperation(testData as RPCRunOperationParam);
 
       const balanceUpdate =
         'metadata' in response.contents[0]
-          ? response.contents[0]['metadata']['balance_updates']
+          ? (response.contents[0]['metadata'][
+              'balance_updates'
+            ] as OperationMetadataBalanceUpdates[])
```

Comment:
was changed because i got a weird error for type conflicts that seemed unreleated to my changes